lib/types.js

```diff
@@ -425,6 +430,14 @@ types.wrap_map = function(m, key_wrapper) {
     }
     return types.Map32(items);
 };
+types.wrap_map_map = function(m) {
```

Comment:
Isn't this just the same as the existing wrap_map, with no second argument?

Reply:
No. You can't iterate over Map contents with for...in nor access contents with []. You have to use the for...of.

Reply:
Makes sense. Minor point, could you rename as wrap_set_as_list and wrap_map_as_map?

Reply:
Done.